db/migrate/20231114155433_add_podcast_apple_config_unique_constraint.rb

```diff
@@ -0,0 +1,5 @@
+class AddPodcastAppleConfigUniqueConstraint < ActiveRecord::Migration[7.0]
+  def change
+    add_index :apple_configs, [:podcast_id], unique: true
```

Comment:
And with rails validations on unique ids across public/private feeds then this transitively applies to the feeds as well `{:podcast_id, :public_feed_id}` etc